Skype/WebSDK/docs/PTAudioP2PEscalation.md

```diff
@@ -0,0 +1,62 @@
+
+# P2P Escalation
+
+ _**Applies to:** Skype for Business 2015_
+
+
+## Escalating a P2P Call to a Group Call
+
+To start off we need to start a P2P Outgoing call, as shown in the **Outgoing P2P** sample.
+Once the call is established, we can add a participant to the conversation which escalates the P2P call to a group call.
+
+In the case of a peer-to-peer, P2P, conversation the server will escalate it to a conference.
+
+```js
+conversation.participants.add('sip:xxx').then(function() {
+    // participant added
+}, function (error) {
+    // error
+});
+```
+
+## Ending a Call
```

Comment:
Maybe highlight difference between this and just stopping the modality? At least mention that both options exist.